omod/src/main/java/org/openmrs/web/dwr/DWRFormService.java

```diff
@@ -109,6 +109,14 @@ public FormFieldListItem getFormField(Integer formFieldId) {
 		for (Field field : Context.getFormService().getFields(txt)) {
 			FieldListItem htmlSafeField = new FieldListItem(field, Context.getLocale());
 			htmlSafeField.setName(Encode.forHtml(htmlSafeField.getName()));
+			htmlSafeField.setFieldTypeName(Encode.forHtml(htmlSafeField.getFieldTypeName()));
+			
+			// Add the description only if it was given
+			String description = htmlSafeField.getDescription();
```

Comment:
The reason for this check was that if a description was not given for a new field (the field is optional), "null" would appear for that row under the `Description` column which looked bad. I guess this was a product of the encoding, but the `if` condition checking that the description was not null before encoding it fixed the issue, allowing for blank values in the `Description` column rather than many "null"s.